libraries/ms-common/inc/bcd.h

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+#include <stdint.h>
+// Binary coded decimal converter for two-digit numbers
+// A value of 0xFF will be returned if an invalid value is passed in
+
+#define INVALID 0xFF

```

Comment:
You should call this something BCD-specific like `INVALID_BCD_ARGUMENT` since it'll pollute the global namespace of anything that includes it, which might want their own `INVALID` macros.